pm.mirko.Amusiz.yml

```diff
@@ -0,0 +1,20 @@
+id: pm.mirko.Amusiz
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '40'
+command: amusiz
+finish-args:
+- "--share=network"
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--socket=pulseaudio"
+- "--socket=wayland"
+- "--env=IS_FLATPAK=1"
+modules:
+- name: amusiz
+  builddir: true
+  buildsystem: meson
+  sources:
+  - type: git
+    url: https://github.com/mirkobrombin/Amusiz
+    tag: 1.3.2
```

Comment:
Plase add a commit aswell. To protect against retagging.

Reply:
can I put both tags and commits in the same module?

Reply:
yes.